pipelines/operator-ci-pipeline.yml

```diff
@@ -207,14 +203,16 @@ spec:
           value: "$(tasks.preflight.results.log_output_file)"
         - name: result_file
           value: "$(tasks.preflight.results.result_output_file)"
+        - name: md5sum
```

Comment:
Could you also add that to the hosted pipeline?

Reply:
Done